ContactsManager/ContactsManager_select_method.html

```diff
@@ -0,0 +1,98 @@
+<div>{{draft}}{{securecontext_header}}{{DefaultAPISidebar("Contact Picker API")}}</div>
+
+<p class="summary">The <strong><code>select()</code></strong> method of the {{domxref("ContactsManager")}} interface returns a {{jsxref('Promise')}} which, when resolved, presents the user with a contact picker which allows them to select contact(s) they wish to share.</p>
+
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="syntaxbox notranslate">var <var>ContactInfo</var> = ContactsManager.select(properties, options);</pre>
+
+<h3 id="Parameters">Parameters</h3>
+
+<dl>
+  <dt><em>properties</em></dt>
+  <dd>An array of {{jsxref('String', 'strings')}} defining what information to retrieve from a contact. Allowed values are as follows:
+    <!-- I'm unsure of this because in reality it's whatever in contained in each of these fields, not the actually name/email etc... -->
+    <ul>
+      <li><code>'name'</code>: The contacts name.</li>
+      <li><code>'tel'</code>: The telephone number(s) of the contact.</li>
+      <li><code>'email'</code>: The email address of the contact.</li>
+      <li><code>'address'</code>: The contacts postal address.</li>
+      <li><code>'icon'</code>: The avatar of the contact.</li>
+    </ul>
+  </dd>
+
+  <dt><code>options</code> {{optional_inline}}</dt>
+  <dd>Options are as follows:
+    <ul>
+      <li><code>multiple</code>: Can be <code>true</code> or <code>false</code>. If set to true allows multiple contacts to be selected. Default is <code>false</code>.</li>
+    </ul>
+  </dd>
+ </dl>
+
+<h3 id="Returns">Return value</h3>
+
+<p>Returns a {{jsxref('Promise')}} which when resolved opens a contact picker.</p>
```

Comment:
Does the promise resolve when the picker opens or when the picker closes? I'd think the latter.

Reply:
It is the latter - changed